docs/data/toolpad/core/components/dashboard-layout/DashboardLayoutBranding.tsx

```diff
@@ -37,19 +49,20 @@ export default function DashboardLayoutBranding() {
   }, [pathname]);
 
   return (
-    <AppProvider navigation={NAVIGATION} branding={BRANDING} router={router}>
+    // preview-start
+    <AppProvider
+      navigation={NAVIGATION}
+      branding={{
```

Comment:
For a follow-up conversation:

Would it make sense to move these into the theme as [custom tokens](https://mui.com/material-ui/experimental-api/css-theme-variables/customization/#adding-new-theme-tokens)?